presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionArgSuite.scala

```diff
@@ -1128,3 +1128,16 @@ class CompletionArgSuite extends BaseCompletionSuite:
       """x: Int
         |x = : Any""".stripMargin,
     )
+
+  @Test def `autofill-arguments-case-class` =
+    check(
+      """
+        |case class A(x: Int, y: Int)
+        |
+        |def main() =
+        |  A(x@@)
+        |""".stripMargin,
+      """x = : Int
+        |x = ???, y = ???Autofill with default values""".stripMargin,
```

Comment:
I don't think we need to old behaviour really, it was very difficult to discover.